docs/apis-tools/community-clients/cli-client/cli-get-started.md

```diff
@@ -5,6 +5,12 @@ sidebar_label: "Getting started with the CLI client"
 description: "Get started with this tutorial that shows you how to interact with Camunda 8 using the community-supported CLI client and command line interface `zbctl`."
 ---
 
+:::note Heads up!
```

Comment:
Do we want this noted all the way up to 8.8, or only 8.6 in the relevant docs?

Reply:
Good question - since this was announced as deprecated in 8.6, any version beyond that should probably have slightly different wording. I could see this being as simple as removing the "now". 

Related - @aleksander-dytko, when the Camunda CLI is ready, should we remove this docs content? Will the GitHub project be archived? How do you want to handle this?

Reply:
@christinaausley keep this as-is. I found time to add these pages to the CCH and remove them from the docs. See https://github.com/camunda/camunda-docs/pull/5147.

Reply:
As-is, as in, keep the changes we have here, or remove these changes from this PR? 😄

Reply:
You can keep the changes we have here, I will remove the pages entirely in the other PR.